includes/compatibility.php

```diff
@@ -333,6 +333,18 @@ public static function save_polylang_meta( $keys, $sync, $from, $to ) {
 	 * @return array Updated post meta.
 	 */
 	public static function on_wp_import_post_meta( $post_meta ) {
+		$wp_importer = get_plugins( '/wordpress-importer' );
+
+		if ( ! empty( $wp_importer ) ) {
+			$wp_importer_version = $wp_importer['wordpress-importer.php']['Version'];
+
+			if ( isset( $wp_importer_version ) && ! empty( $wp_importer_version ) ) {
```

Comment:
```suggestion
			if ( ! empty( $wp_importer_version ) ) {
```
`! empty` includes `isset`.